src/System.Data.Common/src/MatchingRefApiCompatBaseline.uapaot.txt

```diff
@@ -0,0 +1,2 @@
+Compat issues with assembly System.Data.Common:
+TypesMustExist : Type 'System.Data.DataCommonEventSource' does not exist in the implementation but it does exist in the contract.
```

Comment:
@corivera this looks like it was made public for uapaot because of reflection issues in https://github.com/dotnet/corefx/pull/22421? Is that still necessary? Could we do the same thing with rd.xml instead?